src/browser-polyfill.js

```diff
@@ -30,17 +37,26 @@ if (typeof browser === "undefined" || Object.getPrototypeOf(browser) !== Object.
      * not exist when accessed, but behaves exactly as an ordinary WeakMap
      * otherwise.
      *
-     * @param {function} createItem
-     *        A function which will be called in order to create the value for any
-     *        key which does not exist, the first time it is accessed. The
-     *        function receives, as its only argument, the key being created.
+     * @template {object} K
+     * @template V
      */
     class DefaultWeakMap extends WeakMap {
+      /**
+       * @param {function(K):V} createItem
+       *        A function which will be called in order to create the value for any
+       *        key which does not exist, the first time it is accessed. The
+       *        function receives, as its only argument, the key being created.
+       * @param {ReadonlyArray<[K, V]>} [items]
+       */
       constructor(createItem, items = undefined) {
         super(items);
         this.createItem = createItem;
       }
 
+      /**
+       * @param {K} key
+       * @returns {V}
+       */
```

Comment:
This is&nbsp;necessary for&nbsp;proper intellisense.

Reply:
Could you elaborate? What's the difference before and after adding this?

The "get" method for WeakMaps has a well-defined meaning, and the lack of annotations will probably not hurt the comprehension of this method, so I prefer the removal of these boilerplate comments.

Reply:
I’ve found a&nbsp;better way to&nbsp;achieve this.